addons_engine/addons-manager/services/addons_service.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+from enum import Enum
+
+import requests
+from db import addons_db
+
+MARKETPLACE_ADDR = os.environ.get("MARKETPLACE_ADDR") or "http://localhost:11102"
+MARKETPLACE_API = f"{MARKETPLACE_ADDR}/api/v1/marketplace/addons"
+
+addons_service = None
+
+
+# TODO: reuse this enum in addons_monitor
+class AddonStatusEnum(Enum):
+    # Allowed statuses for user to specify.
+    INSTALL = "install"
+    DISABLE = "disable"
```

Comment:
What I have in mind is that the user can issue either disable or install instructions, and when actually the addon is not running anymore because the `addons_monitor` stopped it, the status shown will be 'disabled'. Same case for the install. So when it's actually running it will be changed to active. 

I would love some feedback here because I have been having doubts about the different statuses and I believe it is still confusing somehow.

Reply:
I could change it to, `installing` & `disabling`, indicating that it is not yet disabled or active.